src/main/java/core/basesyntax/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String path) {
+        List<String> operations;
+        try {
+            operations = Files.readAllLines(Path.of(path));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file in: " + path, e);
+        }
+        return operations.subList(1, operations.size());

```

Comment:
The operations.subList(1, operations.size()) method is being used without any checks. This may cause an IndexOutOfBoundsException if the file at the given path is empty or contains only one line. It is recommended to check the size of the operations list before using the sublist method.